### PR TITLE
Impl running build feed and notification for build start

### PR DESCRIPTION
**Pull request by ChouhanSeq**
### Description
* In-progress build feed
* reduce clicks to change dashboard view.

Screenshot 2024-06-13 at 12 33 42 PM

Screenshot 2024-06-13 at 12 32 54 PM

### CODE REVIEW

File: src/Dashboard.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const Dashboard = ({
   const blockToRender = dashboards.find(({ id }) => id === activeTab);
 
   const fetchRunningBuilds = () => {
-    fetch("http://localhost:3003/building")
+    fetch(isProd ? "/building" : "http://localhost:3003/building")
```
Comment (burhan-seq) [inline src/Dashboard.jsx:28]: we can keep it `/building` only for all, no? Why to have a condition.

File: src/Dashboard.jsx [hunk 2]
```diff
@@ -32,16 +32,22 @@ export const Dashboard = ({
...
   const raiseNotifiction = () => {
-    runningBuilds?.map( build => {
-      const id = build?.id;
-      if(!notifiedBuilds[id] && !build?.name?.tolowerCase()?.contains('-pr-')){
-       new Notification(`✅ ${build?.serviceName || ''} Build started`, { body: `${build?.name} in-progress` })
+    runningBuilds?.map((build) => {
+      try {
+        const {id, buildName } = build;
+        if ( !notifiedBuilds[id] && !((buildName || '').includes("-PR-")) ) {
+          new Notification(`✅ ${build?.serviceName || ""} Build started`, {
```
Comment (burhan-seq) [inline src/Dashboard.jsx:39]: The icon is misleading.

File: src/Dashboard.jsx (hunk 1 above)
Comment (ChouhanSeq) [reply src/Dashboard.jsx:28]: Relative url is only applicable in prod build when running the application with yarn dev it needs the complete url.

File: src/Dashboard.jsx (hunk 2 above)
Comment (ChouhanSeq) [reply src/Dashboard.jsx:39]: Updating this icon with 🏁

**Review by burhan-seq — APPROVED** [review]
LGTM

**Pull request merged**